src/w2grid.js

```diff
@@ -315,17 +315,41 @@
         },
 
         operators: { // for search fields
-            "text"    : ['is', 'begins', 'contains', 'ends'],
-            "number"  : ['is', 'between', 'less:less than', 'more:more than'],
-            "date"    : ['is', 'between', 'less:before', 'more:after'],
-            "list"    : ['is'],
-            "enum"    : ['in', 'not in']
+            "text"    : [['is'], ['begins'], ['contains'], ['ends']],
```

Comment:
I would prefer if you kept the same definition as it was - single array. It is more readable. In case when operator and text for it does not match we can use array. Something like
```
            "text"    : ['is', 'begins', 'contains', 'ends'],
            "number"  : ['is', 'between', ['less', 'less than'], ['more', 'more than']],
```